tfac/figures/figure3.py

```diff
@@ -17,3 +19,14 @@ def clustergram_proteins():
     decreased_proteins = proteins.loc[((-0.001>=proteins).any(1) | (proteins>= 0.001).any(1))]
     g = sns.clustermap(decreased_proteins, cmap="bwr", method="centroid", center=0, figsize=(14, 20))
     plt.savefig("output/clustergram_proteins.svg")
+
+def gene_module_enrichm_plot():
+    """ Plots the gene module analysis in heatmap, in the tfac/output folder as modules_v_components.svg """
+    ns, data = ns_RNAseq_data()
+    # z-scoring each column of the data
+    for col in data.columns[1:]:
+        data[col] = (data[col] - np.mean(data[col])) / np.std(data[col])
```

Comment:
Do you mean you're z-scoring the genes before module analysis, or the modules after the analysis? It looks like you're doing it after the analysis here...

Reply:
I did it before.. I divided the code into few functions, the `ns_RNAseq_data()` returns the imported data and the namespaces that `run_module()` will use. I z-scored before passing to the run_module().. is it wrong?

Reply:
We'll you definitely don't want to z-score after running the module analysis. I agree the results here don't look right. Maybe check with @JacksonLChin about what he did?